bears/js/ESLintBear.py

```diff
@@ -48,7 +48,7 @@ def generate_config(filename, file):
         return '{"extends": "eslint:recommended"}'
 
     def process_output(self, output, filename, file):
-        if not file:
+        if not file or not output:
             return
```

Comment:
IMO we'll rather want to trow a warning in that case with a nice message what's wrong instead of silently approving that all code is good!